app/lib/package/model_properties.dart

```diff
@@ -173,6 +173,7 @@ class Pubspec {
   }
 
   late final List<Uri> funding = _inner.funding ?? const <Uri>[];
+  late final hasTopic = topics != null && topics!.isNotEmpty;
```

Comment:
Could be
```suggestion
  late final hasTopic = topics?.isNotEmpty ?? false;
```
To avoid null-assert